src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx

```diff
@@ -81,6 +93,7 @@ export class VanguardArtistWrapper extends React.Component<
           background={backgroundColor}
           pt={50}
           mb={isExpanded && 100}
+          id={slugify(article.title)}
```

Comment:
I would consider putting this in a helper method like `const slugifiedTitle = (title)=> return {slugify(title)}` since we are reusing  `slugify(article.title)` in several places.

Reply:
We are only using it once within render so going to leave this for now

Reply:
It's being used a couple of times in `SeriesWrapper`